### PR TITLE
add se,sh to the su and st

### PR DESCRIPTION
**Pull request by reza1615**
like بزرگترین‌هایشان or بزرگترهایشان

### CODE REVIEW

File: src/affixes [hunk 1]
```diff
@@ -172,14 +172,14 @@ SFX sr      0        ند/FFWW        .
 
 
 SFX st      N        2
-SFX st      0        تر/sdsl        .
-SFX st      0        ترین/sg        .
+SFX st      0        تر/sdslse        .
+SFX st      0        ترین/sgse        .
```
Comment (m-o-s-t-a-f-a) [inline src/affixes]: I think Here 'se' should change with 'sh'. like:
برترین‌هایم 
برترینهایم supposed to be wrong.
could you please update test folder. you can add new file to put some examples for new changes. 
Thanks

File: src/affixes [hunk 2]
```diff
@@ -172,14 +172,14 @@ SFX sr      0        ند/FFWW        .
...
 
 
 ## with zwnj
 SFX su      N        2
-SFX su      0        ‌تر/sdsl       .
-SFX su      0        ‌ترین/sg       .
+SFX su      0        ‌تر/sdslse       .
+SFX su      0        ‌ترین/sgsh       .
```
Comment (m-o-s-t-a-f-a) [inline src/affixes]: here also is same

Comment (reza1615) [inline src/affixes]: it is ok as I mentioned above

File: src/affixes (hunk 1 above)
Comment (reza1615) [inline src/affixes]: for «تر» shouldn't add ZWNJ so 175 is ok but 176 you are right

**Pull request closed without merging**